src/ppx/log.cpp

```diff
@@ -130,7 +134,7 @@ bool Log::CreateObjects(uint32_t modes, const char* filePath)
     //// Allocate mutex dynamically since it doesn't like being
     //// initialized before main...and lock will crash.
     ////
-    //mWriteMutex = std::make_unique<std::mutex>();
```

Comment:
Not part of the CL, but since clang-format touches it, might be a good occasion to remove this comment and dead-code 😊